### ISSUE TITLE
Feature request: Add some support for older iPhone maxed out at iOS 9.3

### ISSUE THREAD

**Issue by hepcat72** [nfarina/homebridge#2374]
### Pre Steps

I have an old iPhone 4s set up as a make-shift smart speaker. It allows me to control homebridge devices using “Hey Siri”. The commands work. The devices receive the command and do the right thing, but Siri always responds that she did not hear back from the device. An iPhone 4s’s maximum supported iOS is 9.3. The iOS cannot be upgraded higher than that.

### Expected Behavior

All I would like to happen is for Siri to not always hang for 10 or so seconds and then say something like “Sorry, I didn’t hear back from your device”.

### Current Behavior

I presume that there’s some older protocol for communicating with the devices, but I don’t know if that’s the case. I imagine that it may have to do with bonjour. Maybe it’s just that the device response is sent before Siri starts listening for it. Or maybe there’s a different format message for device state response.

### Steps to Reproduce (for bugs)

I have 1 iPhone 4s. It syncs its HomeKit data with my other newer iOS devices via settings under iCloud / HomeKit. It issues errors when you do it or says it’s not supported, but it actually works, aside from the side-effects described here. You have to turn on “hey Siri”, but that’s not necessary for testing. Just activate Siri. Commands are the same as you would issue to any other Siri device. And I presume anyone else with an iPhone 4s would get the same erroneous Verbal response I always get.

### Versions

- **Homebridge**: `homebridge --version`
- **Node**:       `node --version`
- **OS**:         

iOS 9.3
I don’t have access to the other versions ATM

### Configuration

```json

```

### Homebridge Log / Command Output

```

```

**Comment by osianSmith** [comment]
This sounds like it may be a limit of iOS 9.3

**Comment by hepcat72** [comment]
Well, it used to work. Something must have changed externally.

**Comment by osianSmith** [comment]
What exactly is the current behaviour? From my understanding Siri does nothing, is this correct?

**Comment by hepcat72** [comment]
1. I say “Hey Siri, turn on the couch fan”. 
2. The couch fan turns on.
3. After a few seconds, Siri says something like “Sorry, I didn’t hear back from your device”.

**Comment by osianSmith** [comment]
Sounds like there is an error in response, did you update homebrew or something?

**Comment by hepcat72** [comment]
Homebrew? None of this involves a Mac computer, so I’m not sure what you mean? Homebridge is installed on an RPi3. It’s been acting this way for quite some time, perhaps since the beginning. My hunch was that the success response either comes back too quickly for the 4s to get it or the bonjour communication protocol was slightly changed since iOS 9.3? I was thinking of adding a sleep in the commander plugin before echoing a true response to see if that worked (if the too-fast theory holds).

**Comment by osianSmith** [comment]
Sorry I meant homebridge (I've been working on homebrew today)

Give it a go and see if it works?
My thoughts are the reply message has changed and it's not being handled correctly

**Comment by hepcat72** [comment]
Well, I suspect that this is a network issue now. I recently issued a config update (simply added an unrelated DHCP reservation for a new security camera) and restarted the Airport Extreme (it's relatively new and supports 802.11ac).  A couple days after that, the first time I issued a command via "Hey Siri" to my iPhone 4S, the following happened:

1. I saw the "Siri Not Available" error on the 4S screen
2. I noted it was connected to my 2.4Ghz network and it said it was not connected to the internet
3. I selected to connect to my 5Ghz network and it then said it was connected
4. Every command I have issued via "Hey Siri" hence:
    - Controlled the powerstate of the device the command was about (which has always worked)
    - Siri responded positively with "done" or "OK" instead of her usual "I tried, but I didn't hear back from your device"

There are other possible factors however: I had recently disabled the 433Mhz receiver because I noted that the Pi was running hot sometimes and sometimes was laggy when the pilight daemon was using 100% of 1 whole processor (50% total).  I learned that this was due to the processing of background 433Mhz signals.  Also, I have noted since then that Siri's response from any device is much faster, the incidence of error responses (i.e. device not responding) is much less, and the necessity of issuing the same command as many as 5 times before it works is much less.

This would have lead me to believe that it was a response speed issue, but since the airport extreme update, I have had other various network-related issues with other devices and apps as well: all my wemo devices, my garage door, my laptop, my mac mini, and my 6S.
